src/App.js

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import './App.css'
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      todos: [
```

Comment:
This looks great style wise. Clearly can see the shape of the data just by scanning those lines 👍